### PR TITLE
Update CDK to latest

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What does this change?

Updates CDK to latest.

The convention is now to have a separate stack per stage. 

## How to test

Deploy to CODE. The application should continue to work as expected.

- [x] Tested in CODE.

### CODE REVIEW

File: cdk/bin/index.ts [hunk 1]
```diff
@@ -1,9 +1,27 @@
 #!/usr/bin/env node
-import { App } from "@aws-cdk/core";
+import { App } from "aws-cdk-lib";
 import { Typerighter } from "../lib";
 
 const app = new App();
 
-new Typerighter(app, "typerighter", {
-  stack: "flexible",
+const stack = "editorial-feeds";
```
[the remark this answers is not in the document]
Comment (jonathonherbert) [reply cdk/bin/index.ts:7]: Ha! Yes 🙃 f4e52dc6

**Comment by rhystmills** [comment]
Not seeing a build for this, do you think we'll need to rebase PRs from before the GitHub Actions CI PR in order to see builds?

**Comment by jonathonherbert** [comment]
@rhystmills I've made a few changes, and things should now work as expected – I'd forgotten to reflect the stage-specific files in the artifact config/riffraff 😅

**Review by rhystmills — APPROVED** [review]
Looks good and successfully deployed to CODE for me.

**Pull request merged**